### PR TITLE
New header sections expanded

### PR DESCRIPTION
**Pull request by zeftilldeath**
## What does this change?
Redesign of the expanded menu state in the new header test.

Simplified expanded sections tab. No account/search/edition switcher present in expanded state now. New close pattern that tested very well with users. 

TODO: Would like or revisit how animation could help make the transition even smoother but will keep that ina separate PR.

## What is the value of this and can you measure success?
New design ready to get the test up and running next week

## Does this affect other platforms - Amp, Apps, etc?
Bloody hope not.

## Before: Closed
old-closed

## After: Closed
new-closed

## Before: Open
old-open

## After: Open
new-open

## Before: Closed slimnav
slim-old-closed

## After: Closed slimnav
slim-new-closed

## Before: Open slimnav
slim-old-open

## After: Open slimnav
slim-new-open

## After: desktop breakpoint
new-desktop

## After: leftcol breakpoint
new-leftcol

### CODE REVIEW

File: static/src/stylesheets/layout/new-header/_menu-item.scss [hunk 1]
```diff
@@ -8,27 +8,9 @@
         @include mq(desktop) {
             float: left;
             margin-right: $gs-gutter;
+            overflow: visible;
```
Comment (zeftilldeath) [inline static/src/stylesheets/layout/new-header/_menu-item.scss]: Allows dividing lines to be visible.

File: static/src/stylesheets/layout/new-header/_new-header.scss [hunk 2]
```diff
@@ -49,8 +53,12 @@ from scrolling */
     // Needs to sit above the menu, and the veggie burger
     z-index: $zindex-main-menu + 2;
 
-    @include mq(tablet) {
-        right: 310px;
+    .new-header--open & {
```
Comment (NataliaLKB) [inline static/src/stylesheets/layout/new-header/_new-header.scss]: What is this for?

Comment (NataliaLKB) [reply static/src/stylesheets/layout/new-header/_new-header.scss]: Answered in person 👍

**Review by NataliaLKB — APPROVED** [review]
Looks good!

**Pull request merged**